app/views/kits/_table.html.erb

```diff
@@ -8,12 +8,12 @@
 </tr>
 </thead>
 <tbody>
-  <% @kits.each do |kit| %>
+  <% @items.each do |item| %>
     <tr>
-      <td><%= kit.name %></td>
+      <td><%= item.name %></td>
       <td>
         <ul>
-          <% kit.line_items.quantities_by_name.map do |id, item_hash| %>
+          <% item.line_items_quantities_by_name.map do |id, item_hash| %>
```

Comment:
@dorner here I am not getting the line_items for item, is this because item is not itemizable, because we added itemizable module to base_item class not item?